vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -304,6 +305,88 @@
             		$('#file').prop("disabled",true);
             		$('#imageId').prop("disabled",false);
             	  });
+				
+            	var currentlyWaiting = false;
+            	var timeout;
+            	$('#slideSpace').sortable({
+	            	update: function(event,ui) {
+		            	var jsonObj = [];
+		            	var blockId;
+		            	$(this).children().each(function(index,element){
+			            	if(typeof($(element.innerHTML).filter('div').attr('data-position'))=="undefined") {
+			            		blockId=$(element).attr('id');
+			            	}else {
+			            		blockId = $(element.innerHTML).filter('div').attr('id');
+			            	}
+			            	if($(element.innerHTML).filter('div').attr('data-position') != index){
+				            	item = {}
+				            	item ["id"] = blockId;
+				            	item ["contentOrder"] = index;
+				            	jsonObj.push(item);
+				            	$(element.innerHTML).filter('div').attr('data-position',index);
+			            	}
+		            	});
+		            	var token = [[${_csrf.token}]];
+		            	if (jsonObj.length != 0){
+			            	if (!currentlyWaiting){
```

Comment:
why do you need this if?  why not just
```
var timeout = null;
...
if (timeout == null) {
   timeout = setTimeout...
} else {
   clearTimeout(timeout);
   timeout = null
}
```
and then make sure to set timeout to null when updateContentOrder is being executed.